mmwtex.c

```diff
@@ -5382,58 +4330,48 @@ print2("</FONT></B></CENTER>\n");
 20-Jun-2015 metamath Google group email:
 https://groups.google.com/g/metamath/c/QE0lwg9f5Ho/m/J8ekl_lH1I8J
 
-There are 3 kinds of section headers, big (####...), medium (#*#*#*...),
-and small (=-=-=-).
+There are 4 kinds of section headers, big (####...), medium (#*#*#*...),
+small (=-=-=-), and tiny (-.-.-.).

```

Comment:
Suspicious: TINY_DECORATION is declared as "-.-." (4 characters).  Using 6 characters in this comment is at least misleading if 4 characters are already sufficient.  Verify.

Reply:
The comment is intended to indicate a repeating unit, as evidenced by the ... in the earlier parentheticals. It's true that you only need 2 repeating units, but the common case is closer to 40 units.